xbmc/Application.cpp

```diff
@@ -5232,6 +5235,12 @@ void CApplication::CloseNetworkShares()
   }
 }
 
+void CApplication::SendGUIMessage(int dwMsg, int senderID/* = 0*/, int controlID/* = 0*/, int param1/* = 0*/, int param2/* = 0*/)
```

Comment:
https://github.com/xbmc/xbmc/blob/master/xbmc/messaging/ApplicationMessenger.cpp#L272

please elaborate why you create an additionsl SendGUIMessage